src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -140,8 +141,75 @@ public static SimulatedPageFile MakeSimulatedPageFileInBookFolder(HtmlDom dom, b
 			return new SimulatedPageFile() {Key = url};
 		}
 
+		/// <summary>
+		/// Producing PDF with gecko doesn't work on Windows for some reason when referencing
+		/// the localhost server.  So we need to use actual files, and refer to actual files in
+		/// the links to the css files.
+		/// </summary>
+		/// <remarks>
+		/// See http://jira.sil.org/browse/BL-932 for details on the bug.
+		/// </remarks>
+		public static SimulatedPageFile MakeRealPublishModeFileInBookFolder(HtmlDom dom)
```

Comment:
It might be simpler to make this return a TempFile...we have a class which SimulatePageFile was based on which is precisely designed to delete the file when it is disposed.